### ISSUE TITLE
Compilation error on `find_vcpkg_root`.

### ISSUE THREAD

**Issue by ln42** [twistedfall/opencv-rust#179]
Running on Linux (nixos), installed opencv4 from the distribution package.

Rustc is 1.47.0, OpenCV4.

When doing `cargo build`, I get:
```
error[E0603]: function `find_vcpkg_root` is private
   --> /home/alois/.cargo/registry/src/github.com-1ecc6299db9ec823/opencv-0.46.2/build.rs:431:40
    |
431 |         let vcpkg_root = canonicalize(vcpkg::find_vcpkg_root(&config)?)?;
    |                                              ^^^^^^^^^^^^^^^ private function
    |
note: the function `find_vcpkg_root` is defined here
   --> /home/alois/.cargo/registry/src/github.com-1ecc6299db9ec823/vcpkg-0.2.6/src/lib.rs:223:1
    |
223 | fn find_vcpkg_root() -> Result<PathBuf, Error> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0061]: this function takes 0 arguments but 1 argument was supplied
   --> /home/alois/.cargo/registry/src/github.com-1ecc6299db9ec823/opencv-0.46.2/build.rs:431:33
    |
431 |         let vcpkg_root = canonicalize(vcpkg::find_vcpkg_root(&config)?)?;
    |                                       ^^^^^^^^^^^^^^^^^^^^^^ ------- supplied 1 argument
    |                                       |
    |                                       expected 0 arguments

error: aborting due to 2 previous errors
```

I see on crates.io that vcpkg wasn't updated in the last 5 months, so I'm not sure where to look?

Thanks for your help

**Comment by twistedfall** [comment]
Should be fixed in v0.46.3, the reason is that the crate is using a vcpkg function that appeared in v0.2.9 and your cached one is probably older. The new crate version should fix that.

**Comment by ln42** [comment]
I can confirm https://github.com/twistedfall/opencv-rust/commit/21a68d02ea9a53d017b83149d7bf751714485cd4 did fix the issue, I just put `0.46.3` explicitly in my `Cargo.toml` and now it works fine (fetching latest 0.2.10 of vcpkg).

Thank you so much for your quick support @twistedfall and for this useful binding!
